tests/test_markers.py

```diff
@@ -9,7 +9,7 @@
 import tomli
 from tasks import MARKER_DEPENDENDENCY_MAP
 
-pytestmarks = pytest.mark.project
```

Comment:
Pretty sure this was me 😅

Reply:
No worries. The purpose of the check will be to catch typos like this. We just went through so many test files, a few typos are inevitable.